zktraffic/tests/test_stats_daemon.py

```diff
@@ -72,4 +89,26 @@ def ping(server, port):
   assert resp.status == 200
   assert "uptime" in resp.read()
 
+  timer.tick()
+  time.sleep(1)
```

Comment:
Why this time.sleep? In practice (mainly code in Aurora) I've found anything actually doing a sleep to be pretty flaky.